app/controllers/account_price_group_members_controller.rb

```diff
@@ -13,7 +13,9 @@ def search_results
     searcher = AccountSearcher.new(params[:search_term], scope: Account.for_facility(current_facility))
 
     if searcher.valid?
-      @accounts = searcher.results
+      @search_results = searcher.results
+      @count = @search_results.count
+      @accounts = @search_results.limit(@limit)
```

Comment:
No need for an ivar here, we're not using `search_results` in the view:
```suggestion
      search_results = searcher.results
      @count = search_results.count
      @accounts = search_results.limit(@limit)
```